Amazon.IonObjectMapper/Serializer/IonListSerializer.cs

```diff
@@ -65,16 +79,17 @@ public override System.Collections.IList Deserialize(IIonReader reader)
                 return typedList;
             }
 
-            throw new NotSupportedException("Don't know how to make a list of type " + listType + " with element type " + elementType);
+            throw new NotSupportedException("Don't know how to make a list of type " + this.listType + " with element type " + this.elementType);
```

Comment:
Can we update all instances of string concatenation with `$"{variable}"` style instead?

Reply:
There were some instances where it was more logical where I didn't update it, but I updated it for all readable strings where we're trying to put a variable in the string to write readable text.